lesson 17/index.js

```diff
@@ -0,0 +1,91 @@
+"use strict"
+// task 1
+const a = 'true';
+const b = 5 > 6;
+const c = 17;
+const d = undefined;
+let k;
+console.log (a);
+console.log (b);
+console.log (c);
+console.log (d);
+console.log (typeof a, typeof b, typeof c, typeof d, typeof k);
+
+// task 2
+let height = 15;
+let width = 20;
+if (height > width) {
+    console.log (height)
+} else {
+    console.log (width)
+}
+
+// task 3
+for (let i = 1; i < 21; i += 1) {
+    if (i % 3 === 0) console.log (i)
+};
+
+// task 4
+let key = true;
+let documents = true;
+let pen = true;
+let apple = true;
+let orange = false;
+if (key && documents && pen && apple || orange) {
```

Comment:
Может и будет true, но сам подсчет неверный.
У && приоритет больше чем ||
Поэтому правильнее будет по условию сделать так
```suggestion
if (key && documents && pen && (apple || orange)) {
```

Reply:
У скобок приоритет выше чем у && и выполнятся они раньше